### PR TITLE
Update bundle_zh_CN.properties

### PR DESCRIPTION
**Pull request by SoMall-dumpling**
The following updates will focus on better unit names, turret names, map names, and other untranslated stuff.

### CODE REVIEW

File: core/assets/bundles/bundle_zh_CN.properties [hunk 1]
```diff
@@ -740,13 +740,13 @@ sector.intersect.name = Intersect
...
-sector.onset.description = 教学区块。区块目标未设立，等待进一步指示
-sector.aegis.description = 敌人被护盾保护。本区块发现一个实验性破盾模块\n定位这个建筑，输入钨并摧毁敌人基地.
-sector.lake.description = 这个区块的熔岩湖限制了大部分单位的移动，悬浮单位可能是唯一可行选择。\n尽快研究这个 [accent]飞船制造厂[]并生产一个 [accent]挣脱
-sector.intersect.description = 初步扫描显示这个区块可能会在登陆后受到来个多个方向的进攻。\n尽快设立防御并扩展。\n[accent]机甲[]单位可用于在这里的崎岖地形活动
-sector.atlas.description = 这个区块包含了各种地形，并需要多兵种配合来快速占领。\n升级单位也许是用于攻击这些更为牢固敌人基地所必须的。\n研究 [accent]电解机[]和[accent]坦克重构厂[]..
-sector.split.description = 这个区块分布着很少敌人，正好可以用来测试新的运输科技
-sector.basin.description = {临时}\n是当前的最高级区块。把这当成一个挑战！ - 更多的区块将在之后发布
+sector.onset.description = 教学区块。区块目标未设立，等待进一步指示。
+sector.aegis.description = 敌人被护盾保护着，不过这里也探测到一台实验性的护盾破坏器。\n找到它，并为它输送一些钨作为弹药，然后摧毁敌方基地。
+sector.lake.description = 这个区块的熔岩湖限制了大部分单位的移动，悬浮单位是唯一选择。\n尽快研究[accent]飞船制造厂[]并生产一个[accent]挣脱[]。
```
Comment (Jackson11500) [inline core/assets/bundles/bundle_zh_CN.properties:745]: 悬浮单位是唯一选择不好，既不符合原文也不符合实际情况

**Comment by xem8k5** [comment]
新的俩词条加一下：

**Comment by xem8k5** [comment]
还有昨天的俩：

**Comment by PCX-LK** [comment]
`mods.github.open-release = 版本页码`
改为“发布页面”更为合适

**Review by xem8k5 — APPROVED** [review]

**Review by Jackson11500 — APPROVED** [review]

**Comment by SoMall-dumpling** [comment]
@Anuken Ready for merge.

**Comment by xem8k5** [comment]
block.heat-source.name = Heat Source
block.heat-source.description = A 1x1 block that gives virtualy infinite heat.

block.heat-source.name = 热量源
block.heat-source.description = 无限输出热量，仅限沙盒。

**Comment by PCX-LK** [comment]
@Anuken Please merge this pr

**Pull request merged**